setup.py

```diff
@@ -70,7 +70,7 @@
 
 setup(
     name="supertokens_python",
-    version="0.16.3",
+    version="0.16.4",
```

Comment:
this is a breaking change PR (see other comment about positional arg). So it should become 0.17.0

Reply:
Moved the positional argument to the end, to avoid the breaking change.